src/pages/edge/adobe-journey-optimizer-decisioning/release-notes.md

```diff
@@ -8,6 +8,12 @@ keywords:
 
 # Release notes
 
+## February 28, 2024
+
+### iOS Optimize 4.0.3
+
+- Added max versions to AEP dependencies in podspec
```

Comment:
This shouldn't say AEP, as that goes against branding guidelines. Change this to "Platform dependencies" and then re-submit.

Reply:
all updated.  i'll close this one out.